client/main/contextmenu/contextmenus.js

```diff
@@ -25,7 +25,7 @@ const createRowMenuItems = function(plural) {
         ]}),
         new RibbonSeparator(),
         new RibbonGroup({ orientation: 'vertical', items: [
-            new ContextMenuButton({ title: _('Delete Row') + (plural ? 's' : ''), name: 'delRow', iconId: 'delrow' })
+            new ContextMenuButton({ title: n_('Delete Row', 'Delete Rows', plural ? 2 : 1), name: 'delRow', iconId: 'delrow' })
```

Comment:
Because this currently only specifies a boolean for plural and the count is not mentioned in the string, it should be written as `title: plural ? n_('Delete Rows') : n_('Delete Row')`